README.md

```diff
@@ -14,7 +14,11 @@ to function correctly.
 Role Variables
 --------------
 
-None
+`address`: IP address or hostname of the iDRAC BMC.
+
+`username`: Username credentials for the iDRAC.
+
+`password`: Password credentials for the iDRAC.
```

Comment:
These are module arguments, rather than role variables.

Reply:
Thanks, fixed this as a priority.